About.html

```diff
@@ -1,122 +1,166 @@
+					>
+					<p
+						>Please contact us per Email for any further questions about Japan Culture Festival
+						2023!</p
+					>
+					<p>japancf2023@culture.org</p>
+				</div>
+			</section>
+			<section id="logo-section">
+				<div id="logo-explain">
+					<div class="title-div">
+						<h1>Japan Culture Festival 2023 Logo</h1>
+						<hr class="hr-bar">
+					</div>
+					<p
+						>The Logo of Japan Culture Festival 2023 was decided through the logo competition from
+						2.February to 3.March.<span class="text-hide2">After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.<br>After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</span></p
+					>
+				</div>
+				<div id="logo-container">
+					<img src="Images/torii (1).png" alt="logo" class="logo" />
+					<div id="logo-subContainer">
+						<h1>Japanese</h1>
+						<h3>Culture Festival 2023</h3>
+						<p>15th-16th Jun 2023 <span id="logo-place">Tokyo, Japan</span></p>
+					</div>
+				</div>
+			</section>
+			<section id="last-festivals">
+				<div class="title-div">
+					<h1>See the past Japan Culture Festival</h1>
+					<hr class="hr-bar">
+				</div>
+				<p
+					>Take a look at the last two Japan culture Festival which took place in Osaka and Kyoto</p
+				>
+				<div id="previous-festivals">
+					<div id="kyoto-festival">
+						<h1>2018</h1>
+						<h3>Japan Culture Festival 2018 in Kyoto</h3>
+					</div>
+					<div id="osaka-festival">
+						<h1>2013</h1>
+						<h3>Japan Culture Festival 2013 in Osaka</h3>
+					</div>
+				</div>
+			</section>
+			<section id="partners" class="hide-about">
+				<div class="title-div">
+					<h1>Partner</h1>
+					<hr class="hr-bar">
+				</div>
+				<div id="partners-container">
+					<div class="partners">
+						<h1>mozilla</h1>
+					</div>
+					<div class="partners">
+						<h1>Google</h1>
+					</div>
+					<div class="partners">
+						<h1>NAVER</h1>
+					</div>
+					<div class="partners">
+						<h1>daumkakao</h1>
+					</div>
+					<div class="partners">
+						<img src="Images/pngwing.com.png" alt="airbnb" class="logo" id="airbnb" />
+					</div>
+				</div>
+			</section>
+		</main>
+		<footer class="style-about">
+			<div id="logo-footer">
+				<img src="Images/torii (1).png" alt="logo" class="logo" />
+				<h1>Japan</h1>
```

Comment:
Please a good HTML semantics rule is that there should be only one h1 tag which should come before all other hx tags on all your pages so as to help in Search engine optimization 👍